_includes/footer.html

```diff
@@ -6,17 +6,17 @@
               <div class="cta-btn mac">
                 <h2>Ready to communicate better?</h2>
                 <a href="{{ site.download_url_mac }}" class="button download-btn" style="color: #{{ page.color }};">Download for Mac</a>
-                <span class="download">For OSX 10.8 or later. <a href="{{ site.download_url_win }}">Download for Windows</a></span>
+                <span class="download">For OSX 10.8 or later. <a href="{{ site.download_url_win }}">Download for Windows</a> <a href="{{ site.download_url_linux }}">Download for Linux</a></span>
               </div>
               <div class="cta-btn win">
                 <h2>Ready to communicate better?</h2>
                 <a href="{{ site.download_url_win }}" class="button download-btn" style="color: #{{ page.color }};">Download for Windows</a>
-                <span class="download">For Windows 7 or later. <a href="{{ site.download_url_mac }}">Download for Mac</a></span>
+                <span class="download">For Windows 7 or later. <a href="{{ site.download_url_mac }}">Download for Mac</a> <a href="{{ site.download_url_linux }}">Download for Linux</a></span>
               </div>
               <div class="cta-btn linux">
                 <h2>We'll let you know when Linux is ready!</h2>
-                <a href="/?platform=linux" class="button download-btn" style="color: #{{ page.color }};">Join Linux Beta</a>
-                <span class="download"><a href="/?platform=win">Download for Mac</a></span>
```

Comment:
We should move these headers out since they should all be the same now, also let's just say download for Mac or Windows, then link to the downloads.